### ISSUE TITLE
Test failures on macOS

### ISSUE THREAD

**Issue by aw1621107** [mockingbirdnest/Principia#1624]
Built on macOS 10.13.1, Apple LLVM version 9.0.0 (clang-900.0.38), target x86_64-apple-darwin17.2.0.

Version built was from commit ab1ca009ab6dacb50ce8812dfff404351e113a27 from eggrobin:use-the-downsampling, with no changes made. The failures have been present for a while, though, so I'm pretty sure they aren't new.

Failed tests:

```
[  FAILED  ] 38 tests, listed below:
[  FAILED  ] EmbeddedExplicitRungeKuttaNyströmIntegratorTest.Restart
[  FAILED  ] DoublePrecisionTest.Consistencies
[  FAILED  ] ЧебышёвSeriesTest.NewhallApproximation
[  FAILED  ] ApsidesTest.ComputeApsidesDiscreteTrajectory
[  FAILED  ] ApsidesTest.ComputeNodes
[  FAILED  ] ElementaryFunctionsTest.ExpLogAndRoots
[  FAILED  ] SymmetricLinearMultistepIntegratorTests/SymmetricLinearMultistepIntegratorTest.LongIntegration/0, where GetParam() = Quinlan1999Order8A
[  FAILED  ] SymmetricLinearMultistepIntegratorTests/SymmetricLinearMultistepIntegratorTest.LongIntegration/1, where GetParam() = Quinlan1999Order8B
[  FAILED  ] SymmetricLinearMultistepIntegratorTests/SymmetricLinearMultistepIntegratorTest.LongIntegration/2, where GetParam() = QuinlanTremaine1990Order8
[  FAILED  ] SymmetricLinearMultistepIntegratorTests/SymmetricLinearMultistepIntegratorTest.LongIntegration/3, where GetParam() = QuinlanTremaine1990Order10
[  FAILED  ] SymmetricLinearMultistepIntegratorTests/SymmetricLinearMultistepIntegratorTest.LongIntegration/4, where GetParam() = QuinlanTremaine1990Order12
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/0, where GetParam() = McLachlanAtela1992Order4Optimal
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/1, where GetParam() = McLachlan1995SB3A4
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/2, where GetParam() = McLachlan1995SB3A5
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/3, where GetParam() = BlanesMoan2002SRKN6B
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/5, where GetParam() = OkunborSkeel1994Order6Method13
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/6, where GetParam() = BlanesMoan2002SRKN11B
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/7, where GetParam() = BlanesMoan2002SRKN14A
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/9, where GetParam() = NewtonDelambreStørmerVerletLeapfrog.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/10, where GetParam() = Ruth1983.AsRungeKuttaNyströmIntegrator<BA>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/11, where GetParam() = Suzuki1990.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/12, where GetParam() = Yoshida1990Order6A.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/13, where GetParam() = Yoshida1990Order6B.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/14, where GetParam() = Yoshida1990Order6C.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/15, where GetParam() = Yoshida1990Order8A.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/16, where GetParam() = Yoshida1990Order8B.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/17, where GetParam() = Yoshida1990Order8C.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/18, where GetParam() = Yoshida1990Order8D.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/19, where GetParam() = Yoshida1990Order8E.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/22, where GetParam() = McLachlanAtela1992Order3Optimal.AsRungeKuttaNyströmIntegrator<BA>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/24, where GetParam() = McLachlan1995SS5.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/25, where GetParam() = McLachlan1995S4.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/26, where GetParam() = McLachlan1995S5.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/27, where GetParam() = McLachlan1995SS9.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/28, where GetParam() = McLachlan1995SS15.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/29, where GetParam() = McLachlan1995SS17.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/30, where GetParam() = BlanesMoan2002S6.AsRungeKuttaNyströmIntegrator<BAB>
[  FAILED  ] SymplecticRungeKuttaNyströmIntegratorTests/SymplecticRungeKuttaNyströmIntegratorTest.LongIntegration/31, where GetParam() = BlanesMoan2002S10.AsRungeKuttaNyströmIntegrator<BAB>
```

Output from `make -j 1 test`:
[principia-tests.log](https://github.com/mockingbirdnest/Principia/files/1464592/principia-tests.log)

**Comment by eggrobin** [comment]
With the exception of `DoublePrecisionTest.Consistencies`, all those tests have libm-dependent tolerances, so the failures are expected (assuming that the tolerances do not change massively).

`DoublePrecisionTest.Consistencies` is worrying; as its name indicate, this checks consistency in somewhat-replicated code in `DoublePrecision`, which does not use libm functions (and strongly depends on things being computed as written).
We are getting:
```
actual   : {+1.99686276774065141e+00, -2.26773229438926904e+00, -2.34507708327514858e-01}|{+2.35135508527109255e-17, -7.12526060033719547e-19, -2.02055570948244712e-18}
expected : {+1.99686276774065141e+00, -2.26773229438926904e+00, -2.34507708327514858e-01}|{+2.35135508527109255e-17, -1.11734828522549374e-16, -2.02055570948244712e-18}
```
The error fails to match in its second coordinate.

**Comment by pleroy** [comment]
Fixed by #2453.
